.github/workflows/tests.yml

```diff
@@ -0,0 +1,71 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 4 * * 1'
+
+jobs:
+  example:
+
+    runs-on: ${{matrix.os}}
+
+    strategy:
+      matrix:
+        os: ["ubuntu-18.04", "windows-2019"]
+        php: ["5.6", "7.0", "7.1", "7.2", "7.3", "7.4"]
+        symfony: ["^3.4", "^4.0", "^5.0"]
```

Comment:
The matrix is IMHO way to big (it's slower + it consumer too much energy for nothing). We do not need to tests all theses combinaison.
Let's reduce the list to 15 elements maximum :pray: